src/Microsoft.DotNet.Build.VstsBuildsApi/JsonHttpClient.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Build.VstsBuildsApi
+{
+    internal class JsonHttpClient
+    {
+        private HttpClient _client;
+
+        public JsonHttpClient(Uri baseAddress, string credentials)
```

Comment:
@chcosta I didn't change the `public` stuff in `internal` classes to `internal` because the class being internal means everything inside will implicitly be internal. If the class ever needs to be public, it reduces the changes needed. Does that sound ok?